zebra-state/src/sled_state.rs

```diff
@@ -42,11 +59,19 @@ impl SledState {
     }
 
     /// Queue a finalized block to be committed to the state.
+    ///
+    /// After queueing a finalized block, call [`process_queue`] to check whether
+    /// the newly queued block (and any of its descendants) can be committed to
+    /// the state.
```

Comment:
should we just do this in `queue`?

Reply:
Sure, that's probably simpler.  I'm not sure when we'd want to queue a block but not process the queue, or when we'd want to process the queue without a new block.

Reply:
Fixed in ea6e3c66